question.php

```diff
@@ -277,8 +277,10 @@ public function is_complete_response(array $response): bool {
      * @return bool whether the two sets of responses are the same - that is
      *                            whether the new set of responses can safely be discarded.
      */
-    public function is_same_response(array $prevresponse, array $newresponse) {
-        return false;
+    public function is_same_response(array $prevresponse, array $newresponse): bool {
+        /* The response has already been filtered against get_expected_data, we just need to filter out attempt state
+           and scoring state before comparing. */
```

Comment:
```suggestion
        // The response has already been filtered against get_expected_data, we just need to filter out attempt state
        // and scoring state before comparing.
```
https://moodledev.io/general/development/policies/codingstyle#inline-comments

Reply:
Ich krieg die Krise, vor einigen Monaten hat irgendeins der moodle-plugin-ci Tools genau das torpediert, weil die erste Zeile dann nicht mit einem Satzzeichen endet :cursing_face: Sonst wäre ich nie auf Block-Comments gekommen.